models/evc.py

```diff
@@ -840,7 +856,15 @@ def deploy_to_path(self, path=None):  # pylint: disable=too-many-branches
         7. Update links caches(primary, current, backup)
 
         """
-        self.remove_current_flows()
+        self.clean_errors('current_path')
+        try:
+            self.remove_current_flows()
+        except EVCPathNotDeleted as err:
+            log.error(f"Error preparing current_path for {self}: {err}")
+            self.deactivate_set_error('current_path', 'delete')
```

Comment:
I liked how `error_status` ended up structured with the path and which specific action it failed for, `delete` in this case. My concern is if we should really expose at this level or abstract one more, for instance, users can't do anything about current_path via API other than enabling/disable or redeploying it, so in this case, if this deletion failed here, users will need to `redeploy`. So, would be more valuable for users to have an action of `redeploy` or is that already understood if the users see a `{"error_status": {"current_path": "delete"}}`?, or should we try to also include "redeploy" in the structure too? Meaning capture, which path, which operation failure, and what the user needs to next? 

There's no right or wrong, as its is, and as a developer familiar with the code, I can understand, but I can also imagine a network operator seeing this and thinking that the current_path needs to be deleted again, so will they understand they need to reploy? Think about it. I'll think about it too.

Reply:
On an actual EVC deletion, jsut the specific action isn't too ambiguous though, since there's only path deletions involved, but during an operation that has both delete/install, then it has room for ambiguity.

Reply:
Change for the value to be an explanation about what happened. E.g. {'current_path': 'Error while deleting old path: In "01:2": Service Unavailable'